### PR TITLE
Fix no frame_size in caps

### PR DESCRIPTION
**Pull request by maksstach**
Don't merge this pull request. Instead make branch `master` the repository base.

The plugin was unusable because `frame_size` and `frame_lengths` were never part of caps.
I haven't dug into it the opus RFC enough to know about frames with length 0 and I suggest not handling this case for now and fixing it when this issue arises and there's a file with an empty frame for testing.

To be released along with https://github.com/membraneframework/membrane_ogg_format/tree/master

### CODE REVIEW

File: lib/membrane_ogg/payloader/opus.ex [hunk 1]
```diff
@@ -17,7 +17,15 @@ defmodule Membrane.Ogg.Payloader.Opus do
   @encapsulation_version 1
   @reference_sample_rate 48_000
 
-  def_options original_sample_rate: [
+  def_options frame_size: [
+                type: :non_neg_integer,
```
Comment (maksstach) [inline lib/membrane_ogg/payloader/opus.ex]: what's the best type here?

Comment (Qizot) [reply lib/membrane_ogg/payloader/opus.ex]: That's the good one :)

File: lib/membrane_ogg/payloader/opus.ex [hunk 2]
```diff
@@ -195,13 +203,8 @@ defmodule Membrane.Ogg.Payloader.Opus do
...
 
-  defp audio_pages(data, ctx, state) do
-    position_offset =
-      granule_position_offset(
-        ctx.pads.input.caps.frame_size,
-        ctx.pads.input.caps.frame_lengths
-        |> Enum.count(fn length -> length != 0 end)
-      )
+  defp audio_pages(data, _ctx, state) do
+    position_offset = div(@reference_sample_rate, 1000) * state.frame_size
```
Comment (Qizot) [inline lib/membrane_ogg/payloader/opus.ex:208]: Isn't this line missing previously present `present_frame_count`? Or is it on purpose?

**Review by Qizot — APPROVED** [review]
Just one question and otherwise LGTM. 

One note, please name PRs more appropriately, this is OSS and many people may be able to see it. A pinch of formality won't hurt you :)

File: lib/membrane_ogg/payloader/opus.ex (hunk 1 above)
Comment (maksstach) [reply lib/membrane_ogg/payloader/opus.ex]: frame size can be 2.5 though

File: lib/membrane_ogg/payloader/opus.ex (hunk 2 above)
Comment (maksstach) [reply lib/membrane_ogg/payloader/opus.ex:208]: On purpose. 

> I haven't dug into it the opus RFC enough to know about frames with length 0 and I suggest not handling this case for now and fixing it when this issue arises and there's a file with an empty frame for testing.

File: lib/membrane_ogg/payloader/opus.ex (hunk 1 above)
Comment (Qizot) [reply lib/membrane_ogg/payloader/opus.ex]: ughh, then go for `float()` I guess

File: lib/membrane_ogg/payloader/opus.ex (hunk 2 above)
Comment (Qizot) [reply lib/membrane_ogg/payloader/opus.ex:208]: It does not quite explain why you got completely got rid of it (I may be missing something here). Shouldn't it use
`length(ctx.pads.input.caps.frame_lengths)`?

Comment (maksstach) [reply lib/membrane_ogg/payloader/opus.ex:208]: `Opus` caps don't contain `frame_lengths` nor `frame_size` and never did - that's the original reason why the plugin didn't work and I made this PR

Comment (Qizot) [reply lib/membrane_ogg/payloader/opus.ex:208]: Sure, sounds good.

**Pull request merged**